### PR TITLE
[15.0][MIG] maintenance_equipment_image: Migration to 15.0

### PR DESCRIPTION
**Pull request by mymage**

### CODE REVIEW

**Review by FernandoRomera — APPROVED** [review]

**Review by AntoniRomera — APPROVED** [review]

**Comment by dreispt** [comment]
Please resolve conflicts.

**Comment by mymage** [comment]
@dreispt done

File: README.md [hunk 1]
```diff
@@ -32,7 +32,6 @@ addon | version | maintainers | summary
 [maintenance_equipment_tags](maintenance_equipment_tags/) | 15.0.1.0.0 | [![etobella](https://github.com/etobella.png?size=30px)](https://github.com/etobella) | Adds category tags to equipment
 [maintenance_plan](maintenance_plan/) | 15.0.1.2.0 |  | Extends preventive maintenance planning
 [maintenance_plan_activity](maintenance_plan_activity/) | 15.0.1.0.2 |  | This module allows defining in the maintenance plan activities that will be created once the maintenance requests are created as a consequence of the plan itself.
-[maintenance_product](maintenance_product/) | 15.0.1.0.0 | [![victoralmau](https://github.com/victoralmau.png?size=30px)](https://github.com/victoralmau) | Maintenance Product
```
Comment (etobella) [inline README.md]: Why are you editing this?

Comment (mymage) [reply README.md]: It gives a conflict, probably I did something wrong. I will try to repair it.

**Review by FrancoMaxime — APPROVED** [review]

**Review by rousseldenis — APPROVED** [review]
Code review

**Review by victoralmau — APPROVED** [review]
Code and functional review OK

**Pull request merged**